archetypes/blog-post/index.md

```diff
@@ -34,6 +34,13 @@ authors:
 tags:
     - change-me
 
+
+# The social copy used to promote this post on Twitter and Linkedin. These
+# properties do not actually create the post and have no effect on the
+# generated blog page. They are here strictly for reference.
```

Comment:
People commonly ask for "good" examples of both of these (as they're slightly different) so they have something to model their own copy by. It might be good to provide links to those examples here. Here are the ones that've been shared with me:

- https://www.linkedin.com/feed/update/urn:li:activity:7171191945841561601
- https://www.linkedin.com/feed/update/urn:li:activity:7169021002394296320
- https://www.linkedin.com/feed/update/urn:li:activity:7155606616455737345
- https://twitter.com/PulumiCorp/status/1763265391042654623
- https://twitter.com/PulumiCorp/status/1762900472489185492
- https://twitter.com/PulumiCorp/status/1755637618631405655